### PR TITLE
dep: replace glide

### PR DESCRIPTION
**Pull request by kopiczko**
Requires https://github.com/giantswarm/operatorkit/pull/128

### CODE REVIEW

File: vendor/github.com/petar/GoLLRB/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
+syntax:glob
```
Comment (kopiczko) [inline vendor/github.com/petar/GoLLRB/.gitignore]: vendor/github.com/google/btree/btree_mem.go imports that and  glide either didn't detect that or was smarter.

File: vendor/github.com/stretchr/testify/.gitignore [hunk 2]
```diff
@@ -1,24 +0,0 @@
-# Compiled Object files, Static and Dynamic libs (Shared Objects)
```
Comment (kopiczko) [inline vendor/github.com/stretchr/testify/.gitignore]: testify was used by tpr package which is gone.

**Comment by xh3b4sd** [comment]
I think it will impact download time as soon as you want to update single dependencies.

**Comment by jgsqware** [comment]
But that is the same with glide. any improvement there?

**Comment by xh3b4sd** [comment]
AFAIK it is not the same. Glide does everything. Dep does only what it needs.

**Comment by jgsqware** [comment]
Ok I get you the wrong way. GOOD!

**Comment by kopiczko** [comment]
I noticed `dep` is slightly faster. But you know I may be excited ;)

Nice feature is you can do `dep ensure -update github.com/giantswarm/apextensions` to update apiextensions only. I don't know how it deals with apiextensions dependencies though.

**Pull request merged**